website/docs: draft of new WS-Fed provider docs (#20091)
goauthentik/authentik: website/docs/add-secure-apps/providers/ws-fed/create-wsfed-provider.md

---

> title: Create a WS-Fed provider

Reviewer: I would update the filename and sidebar entry names too

---

> An authentik WS-Fed provider can be created either from scratch or by using for a WS-Fed provider [metadata](#create-a-ws-fed-provider-from-sp-metadata-import-sp-metadata) exported from the Security Token Service (STS).

Reviewer: ```suggestion
An authentik WS-Federation provider can either be created manually, or by importing WS-Federation [metadata](#create-a-ws-fed-provider-from-sp-metadata-import-sp-metadata) from a Service Provider (SP)
```

> ## Create a WS-Fed provider and application pair

Reviewer: ```suggestion
## Create a WS-Federation provider and application pair
```

5. (Optional) Edit the created WS-Fed provider and configure any further settings.

## Export authentik WS-Fed provider metadata

After an authentik WS-Fed provider has been created via any of the above methods, you can access its metadata in one of two ways:

### Download authentik metadata for a WS-Fed provider

To download the metadata of an authentik WS-Fed provider, follow these steps:

1. Log in to authentik as an administrator, and open the authentik Admin interface.
2. Navigate to **Applications > Providers**.
3. Click the name of the provider you want metadata from. to open its overview tab.
4. On the **Overview** tab, in the **Related objects** section, click on **Download** under **Metadata**. This will download the metadata XML file for that provider.

### Access metadata tab for a WS-Fed provider

To view and optionally download the metadata of an authentik WS-Fed provider, follow these steps:

1. Log in to authentik as an administrator, and open the authentik Admin interface.
2. Navigate to **Applications > Providers**.
3. Click the name of the provider you want metadata from to open its overview tab.

> 4. Navigate to the **Metadata** tab.

Reviewer: ```suggestion
### Access metadata tab for a WS-Federation provider

To view and optionally download the metadata of an authentik WS-Federation provider, follow these steps:

1. Log in to authentik as an administrator, and open the authentik Admin interface.
2. Navigate to **Applications** > **Providers**.
3. Click the name of the provider you want metadata from to open its overview tab.
4. Navigate to the **Metadata** tab.
```
